src/Meta.py

```diff
@@ -0,0 +1,48 @@
+
+from BaseClasses import Tutorial
+from worlds.AutoWorld import World, WebWorld
+from .hooks.Data import hook_early_modify_world_properties
+from .Data import meta_table
+
+enable_region_diagram = meta_table.get("enable_region_diagram", False)
+
+def set_world_attributes(world_properties: dict) -> dict:
+    world_properties['description'] = set_world_doc(world_properties.get('description'))
+    world_properties['webworld'] = set_world_webworld(world_properties.get('webworld'))
+    world_properties = hook_early_modify_world_properties(world_properties)
```

Comment:
I think this hook, and the method it's in, are a source of confusion for me, and my other comments were kinda getting at this. 

So, with the new meta.json, world creators can define content for their WebWorld. That makes sense and is awesome 😃  But what's the use case for having a hook for these properties?

Reasons I ask:
- Hooks are usually a way for a world creator to interact with steps in generation that they don't typically have access to. But meta.json is something they have access to fully, and there's already a hook for after the meta file is loaded in Data. (And other json files have a `before_whatever` hook in their separate files, so that'd make sense for Meta too, even if redundant.) So they already have the ability to define it upfront and to modify it programmatically before it's used.
- A use case for hooks is often getting access to options to dynamically change values based on the option values. But `set_world_attributes` is called before generation starts, so that's not a use case here.

So, without the two use cases above, I'm wondering if we need this hook.

And if we don't need this hook here, we don't need a set world attributes function to dynamically set props on World, right? This file could just format things with newlines and what not, and give the world doc and webworld back to init like Locations gives location_id_to_name or Items gives item_id_to_item or similar, via the import in init. 😃

Reply:
I added a a hook because some people might want to use the python classes and do some minimal dynamic changes
I personally used it to include the version of my manual but now that you remind me that I added the 'after_load_meta_file' that could do just that ill probably remove the hook and do like you said

Reply:
I could convert set_word_doc and set_worlds_webworld to act like location_id_to_name but I would need to move the webworld def and doc to meta.py from \_\_init\_\_.py

Reply:
Yeah, functions in the file are fine for organizing. Mainly just having the file create the values for import, which it does now. 😃 

Changes look great! Very clean.